lib/cea/mp4_cea_parser.js

```diff
@@ -94,6 +103,17 @@ shaka.cea.Mp4CeaParser = class {
               box.reader, box.version);
           timescales.push(parsedMDHDBox.timescale);
         })
+        .box('minf', Mp4Parser.children)
+        .box('stbl', Mp4Parser.children)
+        .fullBox('stsd', Mp4Parser.sampleDescription)
+
+        // These are the various boxes that signal a codec.
+        // Use these to set bistreamFormat_.
+        .box('avc1', bitstreamFormatSetter(BitstreamFormat.H264))
```

Comment:
And avc3?

Reply:
Done.  Also added an h265 alternative (hvc1) and a dolby alternative (dvhe).